### PR TITLE
TST DTCWT-IDTCWT autodiff

### LINKED ISSUE #19: Test gradient of Direct composed with Forward
Example

```python
N = 2**10
x = torch.zeros(1, 1, N, requires_grad=True)
J = 8

dtcwt = murenn.DTCWT(J=J)
idtcwt = murenn.DTCWTInverse(J=J)

x_phi, x_psis = dtcwt(x)
y = idtcwt((x_phi, x_psis))
y0 = y[0, 0, N//2]
y0.backward()
```

Check that `x.grad` is a Dirac ☺️

### PR DESCRIPTION
**Pull request by lostanlen**
closes #19 
multichannel, both `normalize=False` and `True`
adds `IDTCWT` as alias to `DTCWTInverse`

### CODE REVIEW

**Review by xir4n — APPROVED** [review]
Thanks for your test and thanks for correcting the code style!
